builder.json

```diff
@@ -1,7 +1,9 @@
 {
   "name": "aws-crt-swift",
   "enabled": true,
+  "needs_compiler": false,
   "build_steps": [
+    "swift build --version",
```

Comment:
Don't we just want `swift --version` here?

Reply:
we actually dont want this at all, we added this when we were testing and forgot to remove. it has now been removed